pkgs/applications/networking/instant-messengers/beeper/default.nix

```diff
@@ -1,11 +1,21 @@
-{ lib, fetchurl, mkDerivation, appimageTools, libsecret, makeWrapper }:
+{ lib
+, fetchurl
+, mkDerivation
+, appimageTools
+, libsecret
+, makeWrapper
+, writeShellApplication
+, curl
+, yq
+, common-updater-scripts
+}:
 let
   pname = "beeper";
-  version = "3.71.16";
+  version = "3.80.17";
   name = "${pname}-${version}";
   src = fetchurl {
-    url = "https://download.todesktop.com/2003241lzgn20jd/beeper-${version}.AppImage";
-    hash = "sha256-Ho5zFmhNzkOmzo/btV+qZfP2GGx5XvV/1JncEKlH4vc=";
+    url = "https://download.todesktop.com/2003241lzgn20jd/beeper-3.80.17-build-231010czwkkgnej.AppImage";
```

Comment:
So strange they changed this but oh well. I also tried using the direct URL mentioned [here](https://github.com/NixOS/nixpkgs/pull/241992#issuecomment-1730304937) but that doesn't seem to be working either..

Reply:
Yes, I was trying to use that as well but it wouldn't work so I gave up on that.